_docs/install-and-upgrade.md

```diff
@@ -210,9 +210,26 @@ Over time, people have come up with recipes to install The Lounge on different
 setups and platforms, with different tooling, etc. These are not officially
 supported (even when hosted on this website), so use them at your own risk:
 
+### Gentoo
+
+The Lounge is available in [GURU overlay](https://gitweb.gentoo.org/repo/proj/guru.git/tree/net-irc/thelounge) (repository). Please follow the [Gentoo documentation](https://wiki.gentoo.org/wiki/Project:GURU/Information_for_End_Users) to enable GURU.
+
+User can customize how to build The Lounge with USE flags (i.e. whether to use sqlite or not).
+
+_With_ default USE flags:
+```
+emerge net-irc/thelounge
+```
+
+_Without_ sqlite database. Store logs in text files only:
+```
+USE='-sqlite' emerge net-irc/thelounge
+```
+
 - [Ansible role using Supervisor](https://github.com/astorije/ansible-thelounge)
 - [Cloudron app](https://cloudron.io/store/io.github.thelounge.html)
 - [LinuxServer.io Docker images](https://github.com/linuxserver/docker-thelounge)
+- [gentoo-thelounge container image](https://github.com/rahilarious/gentoo-containers/tree/main/gentoo-thelounge) - offers CPU optimizations for better performance
```

Comment:
so... I don't think we need to carry this. We have working containers and I'd rather have people use ours.
It's easier to support that way.